src/partition.jl

```diff
@@ -24,10 +24,12 @@ function EmpiricalVariogram(partition::Partition, var₁::Symbol, var₂::Symbol
 end
 
 """
-    DirectionalVariogram(direction, data, var₁, var₂=var₁; dtol=1e-6, [parameters])
+    DirectionalVariogram(direction, data, var₁::Symbol, var₂=var₁; dtol=1e-6, [parameters])
```

Comment:
I am wondering how we could avoid adding type annotations everywhere and still be clear. For someone using the stack, these arguments should be clear, but I agree with you that first time users may be lost without type annotations.

Reply:
Actually, this isn't needed. I reference below that `var`s are symbols!